varya/README.md

```diff
@@ -0,0 +1,127 @@
+
+Check out this Codepen for a lo-fi example: [https://codepen.io/allancole/pen/zYGNWBJ](https://codepen.io/allancole/pen/zYGNWBJ) (make sure to resize the screen to see the effect in action).
+
+Customizing the responsive content width of a Varya Child Theme is also just as simple as changing a few variables. However, we can’t use CSS-variables directly in `@media` queries ([see here](https://www.w3.org/TR/css-variables-1/#using-variables)). Instead, Varya child-themes come with a `responsive.scss` file that overwrites _only_ the responsive styles of the Varya parent theme when compiled. 
+
+Here are the available variables for changing content widths in `responsive.scss`. 
+
+| Screensize        | Breakpoint Variable | Default Size | Description                                                                                                       |
+|-------------------|---------------------|--------------|-------------------------------------------------------------------------------------------------------------------|
+| **Mobile & down** | `$flexwidth`        | 100%         |                                                                                                                   |
+| **Mobile & up**   | `$breakpoint_sm`    | 560px        |                                                                                                                   |
+| **Tablet & up**   | `$breakpoint_md`    | 640px        | This variables matters the most. It should always match the `$content_width` PHP variable set in `functions.php`. |
+| **Laptop & up**   | `$breakpoint_lg`    | 750px        |                                                                                                                   |
+| **Desktop & up**  | `$breakpoint_xl`    | 1024px       |                                                                                                                   |
+| **Wide & up**     | `$breakpoint_xxl`   | 1280px       |                                                                                                                   |
+
+Once you’ve set these variables to your liking in `/varya-child-advanced/responsive.scss`, run `npm run build` on the child-theme which will recompile `responsive.scss` and reset the content width CSS-variables in `responsive.css`.
+
+## Todos
+ - [DONE] Explore responsive-logic overrides in the child theme.
+ - [DONE] Introduce an advanced child theme example that uses block styles
+ - Audit Varya system for excessive overridden rules
+ - Integrate with the child-theme build tool.
+ - Add Jetpack support?
+ - Optimize responsive font-size styles.
+ - [DONE] Audit how variables appear in the editor, the customizer, and the frontend.
+ - Consider adding defaults fallbacks to top-level css-variables.
+ - Create a demo.
+ - Add a CSS-Variables polyfill or other fallback solution for older browsers.
+ - [WIP] Create a ReadMe that describes how to use it.
+
+### License
+
+- GPL 2.0
```

Comment:
This should link to the LICENSE file. Also, are there any other license disclosures that we should list here?

Reply:
Actually... I think these are all missing license files. 😄

Reply:
Ahh, will do. 

May need to reference the image used in the screenshot here. Need to double check that it’s CC0, first but that would likely go here. I’ll compare with a few themes to see what else might need to be included here.